ec3730/Info.plist

```diff
@@ -85,5 +85,7 @@
 			<dict/>
 		</dict>
 	</array>
+    <key>aps-connection-initiate</key>
```

Comment:
Is this necessary?

Reply:
Because we don't have your team and provision profile, as a result, we were facing crashes and complication during app's build.